src/components/dashboard/dashboard-list/DashboardList.tsx

```diff
@@ -54,12 +57,55 @@ class DashboardList extends Component<DashboardListProps, DashboardListState> {
             role: role,
             type: type,
             status: status,
+            email: email,
             screeningGrade: screeningGrade,
             interviewGrade: interviewGrade,
         });
     };
 
-    closeModal = (): void => {
+    closeModal = (type: string, email: string): void => {
```

Comment:
This looks good to me, please just change the url to reflect the suggested server changes!